sentry_streams/sentry_streams/pipeline/chain.py

```diff
@@ -190,6 +193,23 @@ def build_step(self, name: str, ctx: Pipeline, previous: Step) -> Step:
         )
 
 
+@dataclass
+class ParquetSerializer(Applier[Message[TIn], bytes]):
```

Comment:
IS this the right type of message ? (`TIn` ) 
It seems to me that `parquet_serializer` expects a sequence